File: EvoHtmlToPdf-v8/Samples/WinForms/VB/EvoHtmlToPdfDemo/HTML_to_PDF/Repeat_HTML_Table_Header_Footer_Description.vb
Imports System.ComponentModel

Imports System.Text

Namespace EvoHtmlToPdfDemo.HTML_to_PDF
	Partial Public Class Repeat_HTML_Table_Header_Footer_Description
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Repeat_HTML_Table_Header_Footer_Description_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			descriptionWebBrowser.Visible = True
			Dim descriptionString As String = System.IO.File.ReadAllText("DemoAppFiles\Input\Descriptions\WinForms\HTML_to_PDF\Repeat_Table_Header_Footer.html")
			descriptionWebBrowser.DocumentText = descriptionString
		End Sub
	End Class
End Namespace

File: EvoHtmlToPdf-v8/Samples/WinForms/CS/EvoHtmlToPdfDemo/HTML_to_PDF/Repeat_HTML_Table_Header_Footer_Description.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;

using System.Text;
using System.Windows.Forms;

namespace EvoHtmlToPdfDemo.HTML_to_PDF
{
    public partial class Repeat_HTML_Table_Header_Footer_Description : UserControl
    {
        public Repeat_HTML_Table_Header_Footer_Description()
        {
            InitializeComponent();
        }

        private void Repeat_HTML_Table_Header_Footer_Description_Load(object sender, EventArgs e)
        {
            descriptionWebBrowser.Visible = true;
            string descriptionString = System.IO.File.ReadAllText(@"DemoAppFiles\Input\Descriptions\WinForms\HTML_to_PDF\Repeat_Table_Header_Footer.html");
            descriptionWebBrowser.DocumentText = descriptionString;
        }
    }
}
